### DISCUSSION TITLE
DietPi-Software | Crowdsec

### DISCUSSION THREAD

**Discussion by cocoflan** [MichaIng/DietPi#5434] [Software Request] [answered]
Outnumbering cybercriminals all together

CrowdSec is a free, open-source and collaborative IPS.
Analyze behaviors, respond to attacks & share signals across the community.

https://crowdsec.net/

https://crowdsec.net/blog/how-to-secure-your-raspberry-pi-with-crowdsec/

How to secure your Raspberry Pi OS with CrowdSec

In this article, I’ll describe how to install the CrowdSec agent and the firewall bouncer directly on Raspberry Pi OS and convert it into a sort of honeypot using endlessh (an ssh tarpit) and a web server which only purpose is for CrowdSec to detect attacks in its logfiles.

Raspberry Pi is a perfect device for this as it’s a cheap way to help out collecting CTI – while hopefully annoying criminals who are lured into wasting time in our ssh tarpit. What’s not to like?

One of the biggest changes in CrowdSec from v1.3.0 has happened behind the scenes as we now produce precompiled binaries for arm and Raspberry Pi OS. 

Theoretically, they should work on all versions of Raspberry Pi but this hasn’t been tested yet. So if you run into any problems please let us know. In this example, I’m using Raspberry Pi OS v11.3 on Raspberry Pi 4.

So far only the firewall bouncers for iptables and nftables are available. If you wish to have other bouncers ported to Raspberry PI OS please let us know. We are very community-driven in this area.

**Comment by Joulinar** [comment, marked as answer]
A duplicate of https://github.com/MichaIng/DietPi/discussions/5033

**Comment by cocoflan** [comment]
Ok didn't saw it, i am running it on dietpi for a couple of days, works great.
